docs/navigate.md

```diff
@@ -20,8 +20,11 @@ The plugin supports these options:
         interactive: false,
         active: false,
         cursor: "move",     // CSS mouse cursor value used when dragging, e.g. "pointer"
-        frameRate: 20,
-        mode: "smart"       // enable smart pan mode
+        frameRate: 60,
```

Comment:
Why did 'frameRate' have to change?

Reply:
The actual default frameRate value is 60, and I did not change the working code. However, the documents do not align with the code.

Here is just the document update. I will split these update to another PR, since now I notice it is not very clear to do it here.